addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -91,6 +91,21 @@ const wSnippetMenu = weSnippetEditor.SnippetsMenu.extend({
         )[0];
         delete themeBodyLineHeightEl.dataset.unit;
         delete themeBodyLineHeightEl.dataset.saveUnit;
+        const themeButtonFontFamilyEl = $html.find("[data-variable='buttons-font']")[0];
+        if (!themeButtonFontFamilyEl) {
+            const rowEl = new DOMParser().parseFromString(`
+                <we-row>
+                    <we-fontfamilypicker data-variable="buttons-font"></we-fontfamilypicker>
```

Comment:
FYI - The tag needs to be closed  separately because we need to parse as HTML (otherwise, the elements do not have a `dataset`). If we use `<we-font-familypicker/>` the parsed DOM becomes:
```xml
<we-fontfamilypicker ...>
    <we-button .../>
</we-fontfamilypicker>
```
:shrug: